frontend/views/containers/chatroom/MessagePoll.vue

```diff
@@ -116,7 +116,7 @@ export default ({
       }
 
       const checkAndSetTimer = () => {
-        if (Date.now() - this.pollData.expires_date_ms) {
+        if (Date.now() - this.pollData.expires_date_ms >= 0) {
```

Comment:
When poll is not expired yet, this calculation evaluates to `-` value which is `truthy`. So adding `>= 0` for the bug-fix.